### PR TITLE
Added SparkPost mail setting

### PR DESCRIPTION
**Pull request by morph85**
Refer to #3589: add sparkpost driver to configuration file MailSetting.php

### CODE REVIEW

**Comment by LukeTowers** [comment]
@morph85 could you add it to the october:env command too?

**Comment by morph85** [comment]
Hi @LukeTowers, I'm not familiar with october:env command.
Just trying to add the settings as Laravel supports SparkPost out of box.

File: modules/system/models/mailsetting/fields.yaml [hunk 1]
```diff
@@ -149,3 +149,12 @@ tabs:
...
             condition: value[ses]
+
+        sparkpost_secret:
+          label: system::lang.mail.sparkpost_secret
+          commentAbove: system::lang.mail.sparkpost_secret_comment
+          tab: system::lang.mail.general
+          trigger:
+            action: show
+            field: send_mode
+            condition: value[sparkpost]
```
Comment (LukeTowers) [inline modules/system/models/mailsetting/fields.yaml]: Indentation is four spaces, not two.

Comment (morph85) [reply modules/system/models/mailsetting/fields.yaml]: Updated indentation for both SES and SparkPost settings.

**Pull request merged**